paper-api/src/main/java/io/papermc/paper/persistence/PersistentDataContainerView.java

```diff
@@ -157,4 +157,11 @@ public interface PersistentDataContainerView {
      * @throws java.io.IOException if we fail to write this container to a byte array
      */
     byte[] serializeToBytes() throws java.io.IOException;
+
+    /**
+     * Returns the size of the data container
+     *
+     * @return The size of the data container
```

Comment:
This can also be written as, renders to the same thing
```suggestion
     * {@return the size of the data container}
```